user_guide_src/source/incoming/routing.rst

```diff
@@ -206,9 +206,24 @@ Closures
 
 You can use an anonymous function, or Closure, as the destination that a route maps to. This function will be
 executed when the user visits that URI. This is handy for quickly executing small tasks, or even just showing
-a simple view:
+a simple view. These are always treated as GET requests.
 
-.. literalinclude:: routing/020.php
+.. literalinclude:: routing/065.php
+
+.. _view-routes:
+
+Views
+=====
+
+If you just want to render a view out that has no logic associated with it, you can use the ``view()`` method.
+This method accepts the name of the view to load as the ``$to`` parameter.
+
+.. literalinclude:: routing/021.php
+
+If you use placeholders within your route, you can access them within the view in a special variable, ``$page``.
```

Comment:
I'm curious about the choice of `$page` over something like "data" or "parameters/params" or the like. Was this some precedence from Symfony UX?

Reply:
No, this portion has nothing to do with Symfony UX. I've been using more view-first pages lately and wanted something nicer, that's all. :) I think I got the name from either NextJS or SvelteKit, but I don't recall. I can change the name to `$params` or whatever we think is best, so there's no worry of conflicting with user data vars this way.

Reply:
I can't imagine what `$page[0]` means from the variable name.

Reply:
Agreed. Maybe it should be called `$segments` or something?

Reply:
I'm good with "segments". It will always be some portion of a URI parsed from the Router right? So that makes sense.

Reply:
I'm not sure if it is the best, but it is a value of a URI segment, so `segments` make sense.